modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/dashboard/LevelChart/hooks/useAccountInformation.ts

```diff
@@ -55,6 +50,14 @@ export default function useAccountInformation() {
 		account?.externalReferenceCode
 	);
 
+	const {
+		data: opportunities,
+		isValidating: isValidatingOpportunities,
+	} = useGetOpportunities(
+		account?.externalReferenceCode &&
+			`?pageSize=200&filter=accountName eq '${account?.externalReferenceCode}'`
```

Comment:
Da forma que esta aqui, caso o externalReferenceCode seja `undefined`, o valor `undefined` sera passado para a funcao.

Reply:
se for passado  a requisicao nao sera feita